google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -830,6 +834,12 @@ public Builder setSessionPoolOption(SessionPoolOptions sessionPoolOptions) {
       return this;
     }
 
+    /** Sets the database role that should be used by this instance. */
```

Comment:
I think this needs a more elaborate Javadoc on what this means, as most users won't understand this automatically. I would suggest something like:

```suggestion
    /**
    * Sets the database role that should be used for connections that are created by this instance.
    * The database role that is used determines the access permissions that a connection has. This
    * can for example be used to create connections that are only permitted to access certain tables.
    */
```